tests/conftest.py

```diff
@@ -10,7 +10,7 @@
 import pytest
 
 import aiomcache
-from aiomcache.helpers import pylibmc_get_flag_handler, pylibmc_set_flag_handler
+from tests.flag_helper import demo_get_flag_handler, demo_set_flag_handler
```

Comment:
Importing from `tests` is a bit awkward, I'd suggest `.flag_helper`, or just include in this file, as it's not imported elsewhere.